dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterConfig.java

```diff
@@ -88,7 +88,7 @@ public class MasterConfig implements Validator {
     private double maxCpuLoadAvg = -1;
     private double reservedMemory = 0.3;
     private Duration failoverInterval = Duration.ofMinutes(10);
-    private boolean killYarnJobWhenTaskFailover = true;
+    private boolean killApplicationWhenTaskFailover = true;
```

Comment:
Just curious, why do we set the default value of `killApplicationWhenTaskFailover ` to `true`? If we set it to `false`, will DS continue to track the running application started before and keep idempotence when failover done?

Reply:
Get your point, now, DS will not keep tracking running application, and directly set status of taskInstance to  `NEED_FAULT_TOLERANCE`, so the default value sets `true`. Maybe the failover logic can be optimized.